packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -41,6 +42,29 @@ export function buildRootObject(vatPowers) {
         repeaters.set(index, vatRepeater);
         return vatRepeater;
       },
+      createNotifier(delaySecs, interval) {
+        assert(
+          Nat(delaySecs) >= 0,
```

Comment:
Nat() already errors if it is negative, so this can be:
```suggestion
          Nat(delaySecs)
```

Reply:
@erights points out that the `assert` around `Nat(delaySecs) >= 0` can't ever be reached. I've changed them to bare `Nat(delaySecs)` next to `assert(Nat(interval) > 0, '... must be a positive integer');`

Reply:
Doesn't look fixed. You got rid of the `>= 0`, but there's still a pointless `assert` with unreachable details around the `Nat` call. Did you forget to push?